bindata/network/ovn-kubernetes/003-rbac-controller.yaml

```diff
@@ -0,0 +1,161 @@
+# We need to create this because the controller expects it to exist, and we're
+# the first openshift-specific code running. Other code will manipulate the
+# namespace later.
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: openshift-infra
+
+---
+# The controller also requires a service account in openshift-infra
+# It will use this manually, regardless of which account is configured in the pod
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: sdn-controller
+  namespace: openshift-infra
+
+---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: sdn-controller
+  namespace: ovn-kubernetes
+
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: openshift-sdn-controller
+rules:
+- apiGroups: [""]
+  resources:
+  - namespaces
```

Comment:
As with the node-side ClusterRole, you need to double-check this against the ovn-kubernetes code.